src/pharmpy/tools/covsearch/tool.py

```diff
@@ -467,8 +470,23 @@ def _make_df_steps(best_model: Model, candidates: List[Candidate]):
     models_dict = {candidate.model.name: candidate.model for candidate in candidates}
     children_count = Counter(candidate.model.parent_model for candidate in candidates)
 
+    # Find if longest forward is also the input for the backwards search
+    # otherwise add an index offset to _make_df_steps_function
+    only_forward = [
+        fc for fc in candidates if fc.steps == tuple() or isinstance(fc.steps[-1], ForwardStep)
+    ]
+    largest_forward_step = max([len(fc.steps) for fc in only_forward])
+    only_forwards_largest = [fc for fc in only_forward if len(fc.steps) == largest_forward_step]
```

Comment:
I think this part is a bit confusing, I don't know if it would help a bit if some variables are renamed? E.g. `only_forwards` -> `forwards_candidates`, `only_forwards_largest` -> `largest_candidates`? Not sure if it helps, to me it's just a bit unclear what kind of objects these lists consist of.

Reply:
renamed the two following your suggestions with a slight twist.
`only_forwards` -> `forward_candidates`
`only_forwards_largest` -> `largest_forward_candidates`